scripts/deploy-live.sh

```diff
@@ -71,7 +71,7 @@ getMergedBranchMultidevName "merged-branches-clean.txt"
 # Compare existing environments and merged branches, delete only if the environment exists
 merged_branch=" ${merged_branch_multidev_names[*]} "
 for env in ${existing_terminus_envs[@]}; do
-  if [[ $merged_branch =~ " $env " ]] && [ "$env" != "terminusma" ] ; then
+  if [[ $merged_branch =~ " $env " ]] && [ "$env" != "sculpin" ] ; then
```

Comment:
This exclusion was put in place while @rachelwhitton was testing the newly minted Terminus Manual page. I'm re-purposing it to keep a version of the Sculpin-built site available in GitHub and Multidev.